tags-delete

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
```

Comment:
Why is this script needed?

Reply:
Just a helper for deleting tags quickly locally. @bstavroulakis added it

Reply:
okay 👍

Reply:
Wait a second; I did not realize this is on the root level of our repository. 
If it were in some scripts folder I would not care that much, but I think we should remove it entirely. It adds no value to our code base, but it is a source of confusion. If you find it super helpful, please move it to a [gist](http://gist.github.com/) and link it in our documentation.

Reply:
Ooh, I agree. I will just delete this. Thanks